Applications/Spire/Include/Spire/Ui/SubMenuItem.hpp

```diff
@@ -1,22 +1,22 @@
-#ifndef SPIRE_SUB_MENU_ITEM_HPP
```

Comment:
File needs to be renamed.

Reply:
I renamed these files and they have already been updated in my local repository. It seems that github didn't update the file name change to the remote repository. I will try it again.

Reply:
I renamed these files and they have already been updated in my local repository. It seems that github didn't update the file name change to the remote repository. I will try it again.